src/playcanvas-anim.js

```diff
@@ -1790,3 +1790,19 @@ AnimationComponent.prototype.unsetBlendSession = function(curveName) {
         curSession.unsetBlend(curveName);
     }
 };
+
+AnimationComponent.prototype.playSubstring = function(substr) {
+    for (var clip of this.animClips) {
```

Comment:
for var of is ES6. playcanvas-anim.js (and playcanvas-gltf.js) need to be kept as ES5 (because it needs to work in all browsers and Babel isn't being used in a build process to generate ES5. Please just use a regular for loop.